zzk/service/hoststate.go

```diff
@@ -219,8 +219,16 @@ func (l *HostStateListener) Spawn(cancel <-chan interface{}, stateID string) {
 			return
 		}
 
+		// set up a listener on container state
+		cstate, err := GetState(l.conn, req)
```

Comment:
Why are you getting the state here?  This whole function's purpose is to look up the state object?  And besides, the problem in the ticket explains that the stats.go code is finding a state object for which there is no docker container.  So essentially this block is a no-op